### ISSUE TITLE
Compilation ARMhf

### ISSUE THREAD

**Issue by SammyRamone** [TAMS-Group/bio_ik#3]
The package does not natively compile on armhf due to the compiler options for x86.
Setting FUNCTION_MULTIVERSIONING  to 0 and removing the following from forward_kinematics.h works.
#include <emmintrin.h>
#include <immintrin.h>
#include <x86intrin.h>

**Comment by v4hn** [comment]
Is this still broken upstream?

**Comment by SammyRamone** [comment]
yes, I accidentally pushed a commit on master which solves this, but it disables some optimization for x86 processors
Currently the master is not  compiling on armhf

**Comment by philipp1234** [comment]
Tried to fix it. 

@SammyRamone Could you test on on ARM?
